keras_core/legacy/saving/serialization.py

```diff
@@ -484,6 +484,12 @@ def deserialize(config, custom_objects=None):
             arg_spec = inspect.getfullargspec(cls.from_config)
             custom_objects = custom_objects or {}
 
+            # TODO(nkovela): Replace during Keras 3.0 release
+            # Replace keras refs with keras_core
```

Comment:
After the swap, the replacement should go the other way

Reply:
Makes sense, fixed the TODO comment.